docs/locations.md

```diff
@@ -18,6 +18,8 @@ This configuration design enables a number of different use cases, including:
 
 ## Limitations / Caveats
 
+- Velero only supports a single set of credentials *per provider*. It's not yet possible to use different credentials for different locations, if they're for the same provider.
```

Comment:
Wonder if it'd be a ton more clear to say `even if`. Otherwise it's fine as is.

Reply:
that wouldn't be accurate though. you can use different credentials for different providers.